packages/docs/src/routes/docs/overview/index.mdx

```diff
@@ -58,8 +58,7 @@ Qwik is a new kind of web framework that can deliver instant loading web applica
   </div>
   <div class="card">
     <h3>Optimized for speed</h3>
-    <p>
-      Qwik has unprecedented performance, offering sub-second full page loads even on mobile
+    <p>Qwik has unprecedented performance, offering sub-second full page loads even on mobile
```

Comment:
with a line break after the `<p>` tag the following error was thrown when running the docs locally:
```
10:07:06 PM [vite] Internal server error: Expected a closing tag for `<p>` (61:5-61:8) before the end of `paragraph`
  Plugin: vite-plugin-qwik-city
  File: /Users/zanettin/development/qwik/qwik/packages/docs/src/routes/docs/overview/index.mdx:60:5
  58 |    </div>
  59 |    <div class="card">
  60 |      <h3>Optimized for speed</h3>
     |       ^
  61 |      <p>Qwik has unprecedented performance, offering sub-second full page loads even on mobile
  62 |        devices. Qwik achieves this by delivering pure HTML, and incrementally loading JS only
```

Reply:
i saw it before, i am not sure. seems like mdx is generating nested <p> for some reason